src/web/services/nationalities.service.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBed } from '@angular/core/testing';
+
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { NationalitiesService } from './nationalities.service';
+
+describe('NationalitiesService', () => {
+  beforeEach(() => TestBed.configureTestingModule({
+    imports: [
+      HttpClientTestingModule,
+    ],
+  }));
+
+  it('should be created', () => {
+    const service: NationalitiesService = TestBed.get(NationalitiesService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should return non-empty list of nationalities', () => {
+    const service: NationalitiesService = TestBed.get(NationalitiesService);
+    expect(service.getNationalities()).toBeTruthy();
```

Comment:
Will this pass the CI? You may need mock `httpRequestService` or the API request will be sent?

Reply:
Seems to be passing the CI, probably since `HttpClientTestingModule` is imported. Do I still need to make any changes?